src/main/resources/application.yml

```diff
@@ -5,17 +5,17 @@ spring:
     rest:
       basePath: /resources
   jpa:
-    database: HSQL
+    database: POSTGRESQL
     # we do not want Hibernate to start making DB calls during Jackson rendering: much better to have that
     # throw an exception so we know we have to update the persistence configuration.
     open-in-view: false
     hibernate.ddl-auto: validate
     properties:
       hibernate:
-        format_sql: false
```

Comment:
Not sure I want to delete this one—it's the default, but it's useful to remind people that it's there.